crates/metaslang/bindings/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+version.workspace = true
+rust-version.workspace = true
+edition.workspace = true
+publish = true
+
+name = "metaslang_bindings"
```

Comment:
We need to add the new crate to `USER_FACING_CRATES` in `crates/infra/cli/src/commands/publish/cargo/mod.rs`. Otherwise, CD would fail.